package.json

```diff
@@ -125,8 +125,8 @@
     "webpack-merge": "4.1.4"
   },
   "engines": {
-    "node": ">= 6.0.0",
-    "npm": ">= 3.0.0"
+    "node": ">= 9.0.0",
```

Comment:
We should raise this to 10 and drop the 9 tests completely on CircleCI and instead run 10/11.